content/pt-br/firewall.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Firewall
+status: Completed
+category: tecnologia
+tags: ["segurança", "", ""]
+---
+
+## O que é
+
+Um firewall é um sistema que filtra o tráfego de rede com base em regras especificadas. Firewalls podem ser hardware, software ou uma combinação dos dois.
+
+## Problema relacionado
+ 
+Por padrão, uma rede permite que qualquer pessoa entre e saia, desde que siga as regras de roteamento da rede. Devido a esse comportamento padrão, proteger uma rede é um desafio. Por exemplo, em uma aplicação bancária baseado em microsserviços, os serviços se comunicam uns com os outros transmitindo dados financeiros altamente confidenciais através da rede. Um ator malicioso pode se infiltrar na rede, interceptar a comunicação e causar danos se não houver um firewall.
```

Comment:
I'm thinking about `... permite que qualquer pessoa entre e saia`, maybe we can improve it to something like: `... permite que qualquer tráfego/dado entre e saia`, what you think?